experimental/examples/bubblebench/cursor-shared-tree/src/appState.ts

```diff
@@ -0,0 +1,126 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { IAppState, makeBubble, randomColor } from "@fluid-example/bubblebench-common";
+import { Anchor, brand, ISharedTree, JsonableTree, moveToDetachedField } from "@fluid-internal/tree";
+import { Client } from "./client";
+import {
+    bubbleSchema,
+    clientSchema,
+    numberSchema,
+    stringSchema,
+} from "./schema";
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
if you are going to nest for files in directories, and opt into having the linter object to imports like this you should add an index.ts file which exports the things you need and import from that.